pkg/controller/cabundleinjector/configmap.go

```diff
@@ -109,6 +109,11 @@ func (bi *configMapCABundleInjector) Sync(ctx context.Context, syncCtx factory.S
 	// make a copy to avoid mutating cache state
 	configMapCopy := configMap.DeepCopy()
 	configMapCopy.Data = map[string]string{api.InjectionDataKey: bi.caBundle}
+	// set the owning-component unless someone else has claimed it.
+	if len(configMapCopy.Annotations[api.OwningJiraComponent]) == 0 {
+		configMapCopy.Annotations[api.OwningJiraComponent] = api.OwningJiraComponent
```

Comment:
Shouldn't it be `configMapCopy.Annotations[apiannotations.OpenShiftComponent]` instead?